test/processors/image_processor_test.rb

```diff
@@ -22,9 +22,9 @@ class ImageProcessorTest < ActiveSupport::TestCase
       }.with_indifferent_access
     }
 
-    it "should resize a local file" do
+    it 'should resize a local file' do
       processor.on_message(msg)
-      processor.result_details[:info][:file].must_match /JPEG/
+      processor.result_details[:info][:format].must_equal 'jpg'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.